assets/js/blocks/product-search/editor.scss

```diff
@@ -0,0 +1,12 @@
+.wc-block-product-search__field.input-control {
+	-webkit-appearance: textfield;
+	color: #828b96 !important;
+	line-height: 1.8;
```

Comment:
I think this line can be removed since this same value is inherited from here:
https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/697/files#diff-697d8495c996d8ba14ad4ce199f04e9fR7